### ISSUE TITLE
Core 2.1.x docker: 'de-DE' short date pattern is 'dd.MM.yy' instead of 'dd.MM.yyyy'

### ISSUE THREAD

**Issue by desdoades** [dotnet/runtime#2252]
# Issue Title

On .NET Core 2.1.x linux docker for culture 'de-DE' the short date pattern is 'dd.MM.yy' instead of the expected 'dd.MM.yyyy'.

# General

The short date pattern for 'de-DE' culture is expected to be 'dd.MM.yyyy' as also written in a note in the [official documentation](https://docs.microsoft.com/en-us/dotnet/api/system.datetime.toshortdatestring?view=netcore-2.1)

The following line of code returns different results when run via Visual Studio debug on Windows vs when run in a linux docker container. 
`(new System.Globalization.CultureInfo("de-DE")).DateTimeFormat.ShortDatePattern`

On Visual Studio Windows 10 without docker:
`"dd.MM.yyyy"`
On linux docker (FROM microsoft/dotnet:2.1-sdk):
`"dd.MM.yy"`

Target framework is in both cases the same in the .csproj:
`<TargetFramework>netcoreapp2.1</TargetFramework>`

Probably not relevant but on Windows I have 
`System.Globalization.CultureInfo.CurrentCulture.EnglishName`
`"German (Germany)"`
while on linux docker I get
`System.Globalization.CultureInfo.CurrentCulture.EnglishName`
`"Invariant Language (Invariant Country)"`

**Comment by scalablecory** [comment]
@tarekgh

**Comment by tarekgh** [comment]
@desdoades this issue has been fixed in version 3.x. you may try either to upgrade or to force the desired date pattern like the following code: 

```C#
            CultureInfo ci = new System.Globalization.CultureInfo("de-DE");
            ci.DateTimeFormat.ShortDatePattern = "dd.MM.yyyy";
```

**Comment by desdoades** [comment]
@tarekgh Thanks. Will this also be fixed in a future 2.1.x release or do I have to upgrade or change the short date pattern manually?

**Comment by tarekgh** [comment]
> Will this also be fixed in a future 2.1.x release

No, the fix is not there. 2.1 now is servicing release which the bar is very high to fix anything there especially if there is an easy workaround for the issue.
